lib/internal/worker/js_transferable.js

```diff
@@ -98,29 +97,33 @@ function markTransferMode(obj, cloneable = false, transferable = false) {
   obj[transfer_mode_private_symbol] = mode;
 }
 
+
+webidl.converters.StructuredCloneOptions = webidl
```

Comment:
this should be called `StructuredSerializeOptions` to match the [spec](https://html.spec.whatwg.org/multipage/web-messaging.html#dom-structuredserializeoptions-transfer)